ui/src/types/modules/connection.type.ts

```diff
@@ -25,4 +25,4 @@ export interface ConnectionState {
   onlineUsers: OnlineUser[];
 }
 
-export type ContentType = 'setting' | 'file-manager';
+export type ContentType = 'setting' | 'file-manager' | '';
```

Comment:
None of the differences provided seems to indicate any significant irregularities, potential issues, or opportunities for optimization. The changes proposed seem reasonable and maintain compatibility with existing TypeScript types definitions while adjusting some elements according to updated requirements.